website/docs/reference/widgets/table/README.md

```diff
@@ -72,171 +83,144 @@ Controls how overflowing contents of the column are handled. When turned on, the
+#### Client side search
 
-For the following examples of binding properties, consider a table called `Table_1` with the table data given below:
+Client side search controls whether the user can search the entire data set for their query, or only what's currently shown in the table widget.
 
-```
-[
-  {
-    "step": "#1",
-    "task": "Drop a table",
-    "status": "✅",
-    "action": ""
-  },
-  {
-    "step": "#2",
-    "task": "Create a query fetch_users with the Mock DB",
-    "status": "--",
-    "action": ""
-  },
-  {
-    "step": "#3",
-    "task": "Bind the query using => fetch_users.data",
-    "status": "--",
-    "action": ""
-  }
-]
-```
+#### Column sorting
 
-And this is how the data looks in the table widget:
+Toggles whether table columns are sort-able. When turned on, users may click column headers to sort the table rows by that column's value. This setting only applies while the app is in View mode (where the app is deployed).
 
-![](/img/table\_sample\_data.png)
+<VideoEmbed host="youtube" videoId="hmi7BaF3jFI" title="Table | Column Sorting" caption="Use this setting to enable sorting rows by column value."/>
 
-**selectedRow**
+#### Visible
 
-A table you have created might contain a large amount of data. But when you want to get the data of a single row of the table, the selectedRow property comes into play.\
-selectedRow is an internal property that fetches the data of the table row that the user selected. You can bind different widgets to each table row using this property.
+This controls the widget's visibility on the app's page. When turned off, the widget doesn't appear in the published app. While editing the app on the Appsmith canvas, widgets with visibility turned off appear translucent.
 
-As you already know by now, you can write JS code anywhere in Appsmith inside \{{\}}., To bind the selected row of the table to another widget, open the property pane of it, and add the code snippet given below:
+You can also use JS code to determine the widget's visibility programmatically. Click on `JS` next to the `Visible` field in the properties pane to write JavaScript code.
+For example, drag a checkbox widget `Checkbox1` onto the canvas and bind it to the table's `Visible` property. To enable the `Visible` when the user checks the checkbox, add the following JavaScript code:
 
 ```
-{{<table_name>.selectedRow}}
+{{Checkbox1.isChecked}}
 ```
 
-Where `<table_name>` is the name of your table.
+When you tick the checkbox, `Visible` is set to `true`, and the table becomes visible in the app.
 
-Drag a text widget onto the canvas and bind it to a selected table row of `Table_1`.\
-Open the property pane of the text widget and add the following snippet to its label:
+<VideoEmbed host="youtube" videoId="Jb5bNVhFoRE" title="Visible" caption="Visible"/>
 
-```
-{{Table_1.selectedRow}}
-```
+#### CSV separator 
```

Comment:
again doesn't need so much explanation or a video.  Just make a mention in the Download part under Table header options